md/bonita-docker-installation.md

```diff
@@ -87,17 +158,89 @@ docker run --name=bonita -h bonita -e "TENANT_LOGIN=tech_user" -e "TENANT_PASSWO
+This optional environment variable is used to enable/disable dynamic authorization checking on Bonita REST API. The default value is true, which will activate dynamic authorization checking.
+
+### HTTP_API
+This optional environment variable is used to enable/disable the Bonita HTTP API. The default value is false, which will deactivate the HTTP API.
+
+### JAVA_OPTS
+This optional environment variable is used to customize JAVA_OPTS. The default value is -Xms1024m -Xmx1024m -XX:MaxPermSize=256m.
+
+### ENSURE_DB_CHECK_AND_CREATION
+This optional environment variable is used to allow/disallow the SQL queries to automatically check and create the databases using the database administrator credentials. The default value is true.
+
+### DB_VENDOR
+This environment variable is automatically set to postgres or mysql if the Bonita container is linked to a PostgreSQL or MySQL database using --link. The default value is h2. It can be overridden if you don't use the --link capability.
+
+### DB_HOST, DB_PORT
+These variables are optional, used in conjunction to configure the bonita image to reach the database instance. There are automatically set if --link is used to run the container.
+
+### DB_NAME, DB_USER, DB_PASS
+These variables are used in conjunction to create a new user, set that user's password, and create the bonita database.
+
+`DB_NAME` default value is bonitadb.
+
+`DB_USER` default value is bonitauser.
+
+`DB_PASS` default value is bonitapass.
+
+### BIZ_DB_NAME, BIZ_DB_USER, BIZ_DB_PASS
+These variables are used in conjunction to create a new user, set that user's password and create the bonita business database.
+
+`BIZ_DB_NAME` default value is businessdb.
+
+`BIZ_DB_USER` default value is businessuser.
+
+`BIZ_DB_PASS` default value is businesspass.
+
+### DB_ADMIN_USER, DB_ADMIN_PASS
+These variables are optional, and used in conjunction to create users and databases through the administrator account used on the database instance.
+
+`DB_ADMIN_USER` if no value is provided, this is automatically set to root with MySQL or postgres with PostgreSQL.
+
+`DB_ADMIN_PASS` if no value is provided, this is automatically set using the value from the linked container: MYSQL_ENV_MYSQL_ROOT_PASSWORD or POSTGRES_ENV_POSTGRES_PASSWORD.
+
+### DB_DROP_EXISTING, BIZ_DB_DROP_EXISTING
+`DB_DROP_EXISTING` and `BIZ_DB_DROP_EXISTING` can be used to drop existing databases in order to reuse an existing database instance.
+
+`DB_DROP_EXISTING` default value is N.
+
+`BIZ_DB_DROP_EXISTING` default value is N.
+
+### BONITA_SERVER_LOGGING_FILE, BONITA_SETUP_LOGGING_FILE
+Since Bonita 7.9 BONITA_SERVER_LOGGING_FILE and BONITA_SETUP_LOGGING_FILE can be used to update logging configuration.
+
+`BONITA_SERVER_LOGGING_FILE` default value is /opt/bonita/BonitaSubscription-${BONITA_VERSION}-tomcat/server/conf/logging.properties.
+
+`BONITA_SETUP_LOGGING_FILE` default value is /opt/bonita/BonitaSubscription-${BONITA_VERSION}-tomcat/setup/logback.xml.
```

Comment:
I guess `-tomcat` is not valid anymore in 7.10... :thinking: How can we be sure?
```suggestion
`BONITA_SETUP_LOGGING_FILE` default value is /opt/bonita/BonitaSubscription-${BONITA_VERSION}/setup/logback.xml.
```

Reply:
to be validated